miner/worker_test.go

```diff
@@ -568,7 +568,7 @@ func testGetSealingWork(t *testing.T, chainConfig *params.ChainConfig, engine co
 		}
 		_, isClique := engine.(*clique.Clique)
 		if !isClique {
-			if len(block.Extra()) != 0 {
+			if len(block.Extra()) != 2 {
```

Comment:
2?

Reply:
The test was wrong in that it required the resulting block to have no extradata set for ethash and post-merge even if we set the extradata to `w.setExtra([]byte{0x01, 0x02})`